src/utils/logger.ts

```diff
@@ -1,11 +1,22 @@
 import chalk from "chalk";
+import { log } from "@serverless/utils/log";
 
-export function log(message: string): void {
+export let isV3 = false;
```

Comment:
We could contain switch between v2 and v3 in terms of logging to this file in order not to "pollute" all constructs with the implementation variations between the versions of serverless.

We could have in this logger file an API similar to v3 (with all log level being different method) and call v3 API or v2 unique way of logging (hereby `console.log`) depending on framework version. We would have to replace `log` calls to this file in code base with corresponding required log level.

WDYT ?

Reply:
👍  done that. It was easily possible except for progresses (where the API difference between Ora and the serverless progress is very different)